packages/ui-text-input/src/TextInput/index.tsx

```diff
@@ -289,6 +289,14 @@ class TextInput extends Component<TextInputProps, TextInputState> {
     const computedStyle = getComputedStyle(element)
     const { width, paddingInlineStart, paddingInlineEnd } = computedStyle
 
+    if (width === 'auto') {
```

Comment:
Can we add here `(width === 'auto' || width === '')` ?
In my case on complex page its empty string instead of "auto" and same problem still exist.

Reply:
Hi @raman-savitski-instructure ! I've made the change in https://github.com/instructure/instructure-ui/pull/1185

Reply:
@ke1k0 thanks a lot for this quick fix!